jcommune-model/src/main/resources/org/jtalks/jcommune/model/entity/TopicDraft.hbm.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2011  JTalks.org Team
+    This library is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2.1 of the License, or (at your option) any later version.
+    This library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public
+    License along with this library; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.jtalks.jcommune.model.entity">
+    <class name="TopicDraft" table="TOPIC_DRAFT">
+        <cache usage="read-write"/>
+        <id name="id" column="ID">
+            <generator class="native"/>
+        </id>
+        <property name="uuid" column="UUID" not-null="true"/>
+        <property name="title" column="TITLE" type="string" not-null="false"/>
+        <property name="content" column="CONTENT" type="text" not-null="false"/>
+        <property name="pollTitle" column="POLL_TITLE" type="string" not-null="false"/>
+        <property name="pollItemsValue" column="POLL_ITEMS_VALUE" type="text" not-null="false"/>
+
+        <property name="lastSaved" column="LAST_SAVED" type="org.joda.time.contrib.hibernate.PersistentDateTime"
+                  not-null="true"/>
+
+        <many-to-one name="topicStarter" column="TOPIC_STARTER_ID" class="org.jtalks.jcommune.model.entity.JCUser"
+                     lazy="false"
```

Comment:
Why is it `lazy="false"`? By default we should be lazy. Cancelling laziness is a rare situation and a comment should be left explaining the reasoning.